package.json

```
@@ -17,7 +17,8 @@
     "lint-check": "tsc --noEmit && eslint \"*/**/*.{js,ts,tsx}\" --max-warnings 0",
     "test": "jest --runInBand --config=jest.config.unit.js",
     "coverage": "jest --runInBand --config=jest.config.unit.js --coverage",
-    "test:watch": "jest --runInBand --config=jest.config.unit.js --coverage --watchAll"
+    "test:watch": "jest --runInBand --config=jest.config.unit.js --coverage --watchAll",
+    "e2e": "jest --runInBand"
```

I think it's better to rename jest.config.js in jest.config.e2e.js, and replace :
```suggestion
    "e2e": "jest --runInBand" --config=jest.config.e2e.js
```
to prevent bad use of this file ;)

---

:+1: